packages/docusaurus-theme-classic/src/theme/hooks/useTheme.ts

```diff
@@ -38,7 +38,7 @@ const storeTheme = (newTheme) => {
 
 const useTheme = (): useThemeReturns => {
   const {
-    colorMode: {disableSwitch = false},
+    colorMode: {disableSwitch = false, respectPrefersColorScheme = true},
```

Comment:
```suggestion
    colorMode: {disableSwitch, respectPrefersColorScheme},
```

Reply:
removing default args in JSX because the theme validation/normalization ensure these attributes are never undefined